server/config.ts

```diff
@@ -136,9 +137,12 @@ export default {
     legalSenderJourney: get('LEGAL_SENDER_PHASE_BANNER_LINK', 'https://eu.surveymonkey.com/r/SendLegalMail'),
     mailRoomJourney: get('MAIL_ROOM_PHASE_BANNER_LINK', 'https://eu.surveymonkey.com/r/CheckRule39Mail'),
   },
-  fileUploadsEnabled: toBoolean(get('FILE_UPLOADS_ENABLED', false)),
   smoketest: {
     msjSecret: get('APP_SMOKETEST_MSJSECRET', null),
     lsjSecret: get('APP_SMOKETEST_LSJSECRET', null),
   },
+  featureFlags: {
+    fileUploadsEnabled: toBoolean(get('FILE_UPLOADS_ENABLED', false)),
```

Comment:
I created a `featureFlags` object to contain all our feature flags (keeps them all together), so moved the existing  `fileUploadsEnabled` into it

Reply:
Interestingly I saw that the backlog ticket about file uploads has been rejected - we should think about removing this feature flag... but nothing to do with this story 😆

Reply:
Yes, definitely - I'll add a ticket for it

Reply:
SLM-232 created